README.md

```diff
@@ -53,19 +53,6 @@ It is also compatible with the [Silverstripe MFA module suite](https://github.co
 
 ## Configuration
 
-### Logout across devices
-
-This module respects the `SilverStripe\Security\RememberLoginHash.logout_across_devices` config setting, which defaults to `true`. This means that the default behaviour is to revoke _all_ a user’s sessions when they log out.
-
-To change this so that logging out will only revoke the session for that one device, use the following config setting:
-
-```yml
-SilverStripe\Security\RememberLoginHash:
-  logout_across_devices: false
-```
-
-**Important:** do not set this value to false if users do not have access to the CMS (or a custom UI where they can revoke sessions). Doing so would make it impossible to a user to revoke a session if they suspect their device has been compromised.
-
```

Comment:
This block of doc still seem mostly relevant. It needs to be edited a bit, but should still appear in the doc.

Reply:
I don't think it should?  The session-manager is disabling the `logout_across_devices` functionality no matter what you set the config to.  That seems like the opposite of respecting the config value?